main/swadge2024.c

```diff
@@ -240,11 +241,16 @@ void app_main(void)
     readAllSettings();
 
     // If test mode was passed
-    if (getTestModePassedSetting())
+    if (getTutorialCompletedSetting())
```

Comment:
D'oh, this already exists! That's what I get for instinctively launching with `-m Main`